server/controller/index.js

```diff
@@ -1,18 +1,20 @@
 require('dotenv').config();
 
 const express = require('express');
-
 const { checkAuth } = require('./middleware/authentication');
 const { protectRoutes } = require('./middleware/protectRoute');
 const adminHandler = require('./accountsMangment/admin');
 const customerHandler = require('./accountsMangment/customer');
 const orderHandler = require('./order');
+const { addCaptain } = require('./accountsMangment/captains/addCaptain');
 
 const router = express.Router();
 
 router.use(checkAuth);
 // the all routes start from here
 
+router.route('/addCaptain')
+  .post(addCaptain);
```

Comment:
this route must in captain folder inside `index.js` and put at the `index.js` of controller 
```
const captainHandler = require('./captain');
router.use(captainHandler)
```